### PR TITLE
Added information on configuration and variants.

### LINKED ISSUE #245: Add backend information to info page
It would be nice to have some information about the server configuration on the html page.

### PR DESCRIPTION
**Pull request by jeromekelleher**
Updates the info page with some extra information. Closes #245 

Also adds some tests for the info page, and fixed some tests that were excluded in test_views.

One issue we have here is that I can't find a way to determine the number of variants in a VariantFile; @bioinformed, is there a straightforward way to do this?

### CODE REVIEW

File: ga4gh/frontend.py [hunk 1]
```diff
@@ -57,36 +58,76 @@ class ServerStatus(object):
...
+        Returns the list of (httpMethod, URL) tuples that this server
+        supports.
+        """
         urls = []
         rules = app.url_map.iter_rules()
         excluded_methods = ('OPTIONS', 'HEAD')
         excluded_rules = (
             '/', '/flask-api/static/<path:filename>',
             '/static/<path:filename>')
+        version = "v{}".format(protocol.version)
```
Comment (dcolligan) [inline ga4gh/frontend.py]: doing this string construction should probably be a classmethod of Version

Comment (jeromekelleher) [inline ga4gh/frontend.py]: Good call.

**Comment by dcolligan** [comment]
Can you paste into the PR comments what the display now looks like?

File: ga4gh/datamodel/variants.py [hunk 2]
```diff
@@ -106,6 +111,9 @@ def __init__(self, randomSeed, numCalls, variantDensity, variantSetId):
         self._updated = now
         self._variantSetId = variantSetId
 
+    def getNumVariants(self):
```
Comment (dcolligan) [inline ga4gh/datamodel/variants.py]: add TODO here

Comment (dcolligan) [inline ga4gh/datamodel/variants.py]: er, this is the SimulatedVariantSet... nevermind

**Comment by jeromekelleher** [comment]
Display now give you:
```
GA4GH reference server 0.1.0a2
Protocol version 0.5.1
Operations availableMethod	Path
POST	/v0.5.1/callsets/search
POST	/v0.5.1/readgroupsets/search
POST	/v0.5.1/reads/search
GET	/v0.5.1/references/<id>
GET	/v0.5.1/references/<id>/bases
POST	/v0.5.1/references/search
GET	/v0.5.1/referencesets/<id>
POST	/v0.5.1/referencesets/search
POST	/v0.5.1/variants/search
POST	/v0.5.1/variantsets/search

Uptime
 Running since 24 seconds ago (15:12:55 13 Mar 2015) 
ConfigurationKey	Value
DEBUG	True
REQUEST_VALIDATION	False
RESPONSE_VALIDATION	False

DataVariantSetId	NumVariants
1000g_remote	0
1000g_2013.wt	22000
1000g_2011.wt	23000
1000g_2013	0
1000g_2011	0
```

It's not exactly pretty, but it's information.

**Comment by dcolligan** [comment]
Great!

**Comment by jeromekelleher** [comment]
OK, fixed the errors and the issue you pointed out @dcolligan.

**Comment by dcolligan** [comment]
+1

**Comment by dcolligan** [comment]
@jeromekelleher can you rebase and resubmit?  I can't merge this automatically

**Comment by jeromekelleher** [comment]
@dcolligan - Done.

**Pull request merged**